src/v/cluster/tests/serialization_rt_test.cc

```diff
@@ -957,32 +957,36 @@ void roundtrip_cmd(Key key, Value value) {
 SEASTAR_THREAD_TEST_CASE(commands_serialization_test) {
     for (int i = 0; i < 50; ++i) {
         roundtrip_cmd<cluster::create_topic_cmd>(
-          tests::random_topic_namespace(),
+          model::random_topic_namespace(),
           cluster::topic_configuration_assignment(
             random_topic_configuration(), random_partition_assignments()));
 
         roundtrip_cmd<cluster::delete_topic_cmd>(
-          tests::random_topic_namespace(), tests::random_topic_namespace());
+          model::random_topic_namespace(), model::random_topic_namespace());
```

Comment:
are these namespace renames intended to be associated with a commit that is moving the helpers?